app/helpers/table_helper.rb

```diff
@@ -4,4 +4,12 @@ def app_table(*args, **kwargs, &block)
       govuk_table(*args, **kwargs, &block)
     end
   end
+
+  def provider_colour(placement)
+    placement.provider_name == I18n.t("placements.schools.placements.not_yet_known") ? "secondary-text" : ""
+  end
+
+  def mentor_colour(placement)
+    placement.mentor_names == I18n.t("placements.schools.placements.not_yet_known") ? "secondary-text" : ""
+  end
```

Comment:
Could we combine these into a single method? Could the method name be more descriptive if we do this?